target/riscv/insn_trans/trans_cheri.c.inc

```diff
@@ -164,6 +164,8 @@ TRANSLATE_CGET(cgettag)
 TRANSLATE_CGET(cgettype)
 TRANSLATE_CGET(cgetsealed)
 
+TRANSLATE_CGET(cloadtags)
```

Comment:
We might want to consider renaming `TRANSLATE_CGET` -> `TRANSLATE_INT_CAP` now that it is used by non-`cget*`-instructions. And the same for `gen_cheri_get`.

Reply:
Done.